code/API_definitions/location-subscriptions.yaml

```diff
@@ -0,0 +1,591 @@
+openapi: 3.0.3
+info:
+  title: Device location subscription event API
+  description: Service Enabling Network Function API which create, retrieve and delete subscriptions for events related to the location of a device
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+security:
+  - oAuth2ClientCredentials: []
+  - three_legged: []
+ 
+servers:
+  - url: '{apiRoot}/{basePath}'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: location-subscription/v0
```

Comment:
We have to choose a more specific name for this API, as in future we are likely to have another location subscription API for geotracking. This one is related to location verification or geofencing use case. There is an open issue to choose this name: #69 
Once chosen, we have to align it here and also in title, descriptions, etc

Reply:
it seems that `geofencing` is the name with more consensus, so you can apply it